src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -0,0 +1,360 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM AdvSIMD hardware instructions via intrinsics
+    /// </summary>
+    [CLSCompliant(false)]
+    public abstract class AdvSimd
```

Comment:
As with https://github.com/dotnet/coreclr/pull/25508, this doesn't include all the "currently implemented" methods.

All of them will require changes anyways (give the naming and layout changes), so this helps keep https://github.com/dotnet/coreclr/pull/25508 smaller and more localized.